quinn-proto/src/connection.rs

```diff
@@ -2743,15 +2741,11 @@ impl Connection {
         self.state.is_drained()
     }
 
-    pub fn accepted_0rtt(&self) -> bool {
```

Comment:
This is a necessary part of the public API for 0-RTT support: higher-level code needs it to determine whether 0-RTT streams still exist following connection establishment.

Reply:
Brought it back!